scripts/control-plane-dpu/on-server/install.sh

```
@@ -0,0 +1,226 @@
+cp "$SCRIPT_DIR/bf.cfg.template"      "$WORK_DIR/bf.cfg"
+chmod 755 "$WORK_DIR/provision-dpu.sh" \
+          "$WORK_DIR/dpuinstall.sh" \
+          "$WORK_DIR/post-power-cycle.sh" \
+          "$WORK_DIR/setup_netplan.sh"
+
+# ── Copy per-node server configs ───────────────────────────────────────────────
+
+if [[ -d "$SCRIPT_DIR/servers" ]]; then
+    log "Copying per-node server configs..."
+    cp -r "$SCRIPT_DIR/servers" "$WORK_DIR/"
+    log "  $(ls "$SCRIPT_DIR/servers" | wc -l | tr -d ' ') node(s) copied"
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)